src/execution/experimental-executions.js

```diff
@@ -487,28 +642,29 @@ rules.addRule(
     createPureMetric
 );
 
-function getMetricFactory(measure) {
-    const factory = rules.match(measure);
+function getMetricFactory(measure, mdObj) {
+    const factory = rules.match(measure, mdObj);
 
     invariant(factory, `Unknown factory for: ${measure}`);
 
     return factory;
 }
 
-const isDateFilterExecutable = dateFilter =>
-    get(dateFilter, 'from') !== undefined &&
-    get(dateFilter, 'to') !== undefined;
-
-const isAttributeFilterExecutable = listAttributeFilter =>
-    notEmpty(get(listAttributeFilter, ['default', 'attributeElements']));
+function isDateFilterExecutable(dateFilter) {
+    const { from, to } = getDateFilterInterval(dateFilter);
+    return from !== undefined && to !== undefined;
+}
 
+function isAttributeFilterExecutable(attributeFilter) {
+    return notEmpty(getAttrFilterElements(attributeFilter));
+}
 
 function getWhere(filters) {
     const executableFilters = filter(
-        filters, ({ listAttributeFilter }) => isAttributeFilterExecutable(listAttributeFilter)
+        filters, attributeFilter => isAttributeFilterExecutable(attributeFilter)
     );
     const attributeFilters = map(executableFilters, attributeFilterToWhere);
-    const dateFilters = map(filter(filters, ({ dateFilter }) => isDateFilterExecutable(dateFilter)), dateFilterToWhere);
+    const dateFilters = map(filter(filters, dateFilter => isDateFilterExecutable(dateFilter)), dateFilterToWhere);
```

Comment:
You don't have to create extra function for wrapping `isDateFilterExecutable`

Reply:
fixed